packages/react-native-hms/src/classes/HMSSDK.tsx

```diff
@@ -1182,6 +1185,29 @@ export class HMSSDK {
     return HMSManager.lowerRemotePeerHand(data);
   };
 
+
+  getPeerListIterator = (options?: HMSPeerListIteratorOptions): HMSPeerListIterator => {
```

Comment:
@stanwolverine since this is a synchronous operation do we need any decorators/declarations here?